pkg/tfbridge/detailed_diff.go

```diff
@@ -154,6 +157,99 @@ func makeBaseDiff(old, new resource.PropertyValue) baseDiff {
 	return undecidedDiff
 }
 
+// validInputsFromPlan returns true if the given plan property value could originate from the given inputs.
+// Under the hood, it walks the plan and the inputs and checks that all differences stem from computed properties.
+// Any differences coming from properties which are not computed will be rejected.
+// Note that we are relying on the fact that the inputs will have defaults already applied.
+// Also note that nested sets will only get matched if they are exactly the same.
+func validInputsFromPlan(
```

Comment:
Since we're confident this code should not handle Secret and Output wrappers why don't we assert that those are not being encountered?